content/en/docs/v3.5/tutorials/how-to-setup-cluster.md

```diff
@@ -17,7 +17,7 @@ NAME_3=machine-3
 HOST_1=10.240.0.17
 HOST_2=10.240.0.18
 HOST_3=10.240.0.19
-CLUSTER=${NAME_1}=http://${HOST_1}:2380,${NAME_2}=http://${HOST_2}:2380,${NAME_3}=http://${HOST_3}:2380
+CLUSTER=http://${HOST_1}:2380,http://${HOST_2}:2380,http://${HOST_3}:2380
```

Comment:
Hi @cfanbo - Can you please explain this change?

The etcd `--initial-cluster` configuration option does require server names, refer https://etcd.io/docs/v3.5/op-guide/configuration/#clustering. 

For example:

```bash
--initial-cluster 'default=http://localhost:2380'
```

Please let me know if I am missing something.